src/shoobx/mocks3/config.py

```diff
@@ -63,7 +63,7 @@ def configure(config_file):
         filename=filename, level=config.get('shoobx:mocks3', 'log-level'))
 
     directory = config.get('shoobx:mocks3', 'directory')
-    models.s3_backends["global"].directory = directory
+    models.s3_backends["12345678910"]["global"].directory = directory
```

Comment:
What is this hardcoded `"12345678910"` string? My gut tells me it should not be hardcoded.

Reply:
because that's what moto has it hardcoded (without any exported variable so far as I can see http://docs.getmoto.org/en/latest/docs/multi_account.html). I've at least certralized it now.